app/controllers/inbox/trash.js

```diff
@@ -1,12 +1,8 @@
 import { inject as service } from '@ember/service';
-import Controller from '@ember/controller';
-import DefaultQueryParamsMixin from 'ember-data-table/mixins/default-query-params';
+import Route from '@ember/routing/route';
 import { action } from '@ember/object';
 
-const CONCEPT_STATUS = 'a1974d071e6a47b69b85313ebdcef9f7';
-export default class InboxTrashController extends Controller.extend(
-  DefaultQueryParamsMixin
-) {
+export default class InboxTrashController extends Route {
   @service currentSession;
   @service store;
   @service router;
```

Comment:
same as the meetings controller, this is now missing params

Reply:
same here, should extend Controller, not Route